### PR TITLE
[stable/fluent-bit] Adding ES auth secret

### PR DESCRIPTION
**Pull request by endrec**
#### What this PR does / why we need it:

This PR makes it possibly to use kubernetes secrets to pass the password for the elasticsearch output plugin.

#### Special notes for your reviewer:
This is a retake on #9374.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

Signed-off-by: Endre Czirbesz <endre.czirbesz@rungway.com>

### CODE REVIEW

File: stable/fluent-bit/templates/config.yaml
Comment (monotek) [inline stable/fluent-bit/templates/config.yaml:114]: password should be in a secret not in a configmap

**Review by monotek — CHANGES_REQUESTED** [review]

File: stable/fluent-bit/templates/config.yaml
Comment (endrec) [reply stable/fluent-bit/templates/config.yaml:114]: And it is in a secret. `.Values.backend.es.http_passwd` is the old option to provide the password in values, which I kept for backward compatibility.
My addition would be the use of `${HTTP_PASSWORD}` environment variable, which is populated from a secret.

**Comment by endrec** [comment]
@monotek Is there anything else I should do to get this merged?

**Comment by monotek** [comment]
I guess you have to wait for on of the maintainers to approve it...

**Comment by endrec** [comment]
@monotek I thought you were one of the maintainers. :)

**Comment by monotek** [comment]
no, just came by by accident ;-)

**Comment by endrec** [comment]
It's a question for @hectorj2f then: do I need to do anything else?

**Pull request merged**
